apps/fabric-website/src/styles/_variables.scss

```diff
@@ -49,7 +49,7 @@ $CodeBlock-toggleButtonHeight: 28px;
 $Header-backgroundColor: $ms-color-black;
 $Header-height: 50px;
 $Header-height-full-uhf: $Header-height * 2;
-$Nav-width: 208px;
```

Comment:
Do we really need to change this?

Reply:
@Jahnp and @micahgodbolt asked for it since there is a problem with one of the category names being too long for it to fit when the scroll bar is shown